CHANGELOG

```diff
@@ -1,3 +1,36 @@
+Changes in cclib-1.8
+
+Features:
+    * [GSoC 2023] New parser: NBO (Weronika Zak, #1230, #1233, #1244)
+    * [GSoC 2023] New parser: GAMESS *.dat (Weronika Zak, #1208, #1214, #1229)
+    * New method: compute CM5 charges (Sai Murali Karthik Putcha, #676, #852, #1136)
+    * Parse atomcoords for more DALTON calculation types (#641, #1237)
+    * Be less restrictive in parsed data attributes required for Open Babel bridge (#641, #1236)
+    * Support parsing implicit solvation parameters for Gaussian, ORCA, QChem, and Turbomole (#1171, #1217, #1232)
+    * Support parsing excited state method for Gaussian, ORCA, and Turbomole (#1171, #1219)
+    * Support parsing post-HF excited states in ORCA (#1168)
+    * Support parsing Hessian from DALTON, GAMESS, NWChem, and Psi4 (#1199, #1200, #1201, #1202)
+    * Support parsing Psi4 1.7 (Dustin Wheeler, #1185, #1210)
+    * Add vibrational frequency attributes to MOLDEN writer (#1127, #1132)
+    * Allow natural orbitals instead of canonical MOs in MOLDEN writer (#948)
+    * Support parsing Hirshfeld and CM5 charges from Gaussian, ORCA, and QChem (#1137)
+
+Bugfixes:
+    * Fix edge case of XYZ files being interpreted as Turbomole outputs (#1207, #1238)
+    * Fix units for calculating nuclear repulsion energy (migatt, #1241, #1242, #1243)
+    * Fix infrastructure for reading CJSON (#1222, #1234)
+    * Fix edge cases when parsing certain Turbomole outputs (#1174, #1197, #1220, #1222)
+    * Fix automatically updating both cclib.github.io and cclib.readthedocs.io and testing docs build (#709, #1154, #1158, #1203, #1216, #1223, #1227)
+    * Fix precision in parsing Hirshfeld charges from ORCA (#1209, #1213)
+    * Fix parsing Hessian from formatted checkpoint files (#1204)
+
+Developer facing:
```

Comment:
Historically we haven't done this.  Users will not largely not care, but maybe it's helpful for us.  What do we think?

Reply:
I think it's nice, but is it obvious what the number refers to?

Reply:
Probably not; I could add a line at the top to the effect of "Numbers after pound sign refer to relevant GitHub issues and pull requests".

Perhaps related, maintaining two changelogs (this "traditional" one and the Sphinx one, which I'll update after this is finalized) seems silly.  Can we only keep the Sphinx one or is there a good reason for both?